lib/services/ios-project-service.ts

```diff
@@ -388,27 +413,75 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			await this.setupSigningForDevice(projectRoot, buildConfig, projectData);
 		}
 
-		if (buildConfig && buildConfig.codeSignIdentity) {
-			args.push(`CODE_SIGN_IDENTITY=${buildConfig.codeSignIdentity}`);
-		}
+		// These are now set in the .pbxproj, avoid passing them as arguments!
+		// if (buildConfig && buildConfig.codeSignIdentity) {
+		// 	args.push(`CODE_SIGN_IDENTITY=${buildConfig.codeSignIdentity}`);
+		// }
 
-		if (buildConfig && buildConfig.mobileProvisionIdentifier) {
-			args.push(`PROVISIONING_PROFILE=${buildConfig.mobileProvisionIdentifier}`);
-		}
+		// if (buildConfig && buildConfig.mobileProvisionIdentifier) {
+		// 	args.push(`PROVISIONING_PROFILE=${buildConfig.mobileProvisionIdentifier}`);
+		// }
 
-		if (buildConfig && buildConfig.teamId) {
-			args.push(`DEVELOPMENT_TEAM=${buildConfig.teamId}`);
-		}
+		// if (buildConfig && buildConfig.teamId) {
+		// 	args.push(`DEVELOPMENT_TEAM=${buildConfig.teamId}`);
+		// }
+
+		await this.xcodebuild(args, projectRoot, buildConfig.buildOutputStdio);
+		await this.createIpa(projectRoot, projectData, buildConfig);
+	}
 
-		// this.$logger.out("xcodebuild...");
-		await this.$childProcess.spawnFromEvent("xcodebuild",
-			args,
+	private async xcodebuild(args: string[], cwd: string, stdio: any = "inherit"): Promise<ISpawnResult> {
+		const localArgs = [...args];
+		const xcodeBuildVersion = await this.getXcodeVersion();
+		try {
+			if (helpers.versionCompare(xcodeBuildVersion, "9.0") >= 0) {
+				localArgs.push("-allowProvisioningUpdates");
+			}
+		} catch (e) {
+			console.warn("Failed to use xcodebuild with -allowProvisioningUpdates due to error: " + e);
+		}
+		if (this.$logger.getLevel() === "INFO") {
+			localArgs.push("-quiet");
+			this.$logger.info(`xcodebuild ${localArgs.join(" ")}`);
```

Comment:
As above

In case you want to print dots, you can use:
```TypeScript
const action = () => this.$childProcess.spawnFromEvent("xcodebuild", ... );
this.$logger.printInfoMessageOnSameLine("Building application...");
await this.$progressIndicator.showProgressIndicator(action(), 2000);
```